app/javascript/packages/document-capture/components/file-input.tsx

```diff
@@ -184,7 +243,16 @@ function FileInput(props, ref) {
    * @param {Blob|string|null|undefined} fileValue File or string for which to generate label.
    * @return {{'aria-label': string} | {'aria-labelledby': string}}
    */
-  function getAriaLabelPropsFromValue(fileLabel, fileValue) {
+
+  type ariaLabelReturnType = {
+    'aria-labelledby'?: string;
+    'aria-label'?: string;
+  };
+
+  function getAriaLabelPropsFromValue(
+    fileLabel: string,
+    fileValue: Blob | string | null | undefined,
+  ): ariaLabelReturnType {
```

Comment:
I guess I hadn't considered inline type declarations being valid, though I'd probably expect this to be toward the top of the file and named as UpperCamelCase. That, or inlining it as it had been.

```suggestion
  function getAriaLabelPropsFromValue(
    fileLabel: string,
    fileValue: Blob | string | null | undefined,
  ): {
    'aria-labelledby'?: string;
    'aria-label'?: string;
  } {
```

It's also not exactly a one-to-one since before it would have been an object that would have _either_ `aria-label` or `aria-labelledby`, but never both. Now it could allow both properties, or neither. Not sure it really matters in the long-run.

Reply:
I think we should get the right, though I think putting this particular return type inline makes the function sig harder to read. I'm going to add the correction in [this subsequent PR](https://github.com/18F/identity-idp/pull/7707) (since the current was already merged).